backend/tracim_backend/__init__.py

```diff
@@ -73,6 +75,23 @@
     warnings.simplefilter("ignore")
 
 
+class TracimPyramidContext(PyramidContext):
+    """
+    Customize the hapic context to avoid committing a transaction when an exception is caught
+    by hapic.
+    """
+
+    def doom_request_transaction(self, exc: Exception, *args, **kwargs) -> None:
+        try:
+            request = next(arg for arg in args if isinstance(arg, Request))
```

Comment:
Curious way to get the request, i do imagine you do not have choice, a comment here to explain may be nice.

Reply:
I understand is as: takes the next argument which is a request.

Reply:
what suprise me is more that we do need to parse the list to get the request instead of getting it directly.

Reply:
Explanation added.

Reply:
It may seems more logical to get an arguements "request" from kwargs, for ex.
But this is not a big complain, feel free to clarify or not.